issuetracker/github/github_test.go

```diff
@@ -0,0 +1,31 @@
+package github
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_IssueTracker_issueAPIOrigin(t *testing.T) {
```

Comment:
After changing the tested func name you'll have to test the testname as well.

Also, could you replicate this test in the other components as well? - pivotaltracker & gitlab

Reply:
done